static/_files/css/level_2013.css

```diff
@@ -2666,7 +2666,7 @@ html{
     padding: 10px 0 10px 10px;
     color:White;
     background-color: black;
-    font-family:'proxima-nova', sans-serif;
+    font-family:Arial, sans-serif;
```

Comment:
Would it be a good idea to add another variable for `Arial, sans-serif` since it's being updated in so many places at once?

Reply:
@cbryant24 We do have a variable for that, but we can't use it here because it's vanilla CSS and not Sass.

Reply:
thanks, didn't realize that @sharissis-brei